bittensor/utils/networking.py

```diff
@@ -23,8 +23,8 @@
 import miniupnpc
 import netaddr
 import requests
-
 from loguru import logger
+import bittensor
```

Comment:
What do you think about import just the value instead of full module here?? (using relative path)

From my point of view, inside of a module we should use relatives paths since it's the module itself where we are